app/components/sections/hero-section.tsx

```diff
@@ -79,7 +79,7 @@ function HeroSection({
         )}
       >
         <div className="flex flex-auto flex-col">
-          <H2 as="p">{title}</H2>
+          <H2 as="h2">{title}</H2>
```

Comment:
Revert this to a heading. It's the title of the page. I'm not sure what I was thinking before.